libraries/AP_Scripting/examples/BQ40Z_bms_shutdown.lua

```diff
@@ -0,0 +1,93 @@
+-- TI BQ40Z BMS shutdown script
+---@diagnostic disable: param-type-mismatch
```

Comment:
This is cheating, we need to fix the issue not ignore. I think you said that we needed to update the docs for `get_device` https://github.com/ArduPilot/ardupilot/pull/27287#discussion_r1674770170

Reply:
I need to look back into it, but I did this as I was grabbing a parameter and the get() function can return nil according to the docs. I'll look into the bindings a bit more and find a less-cheaty way.